<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFileSystemHorizontalWnd</class>
 <widget class="QWidget" name="CFileSystemHorizontalWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>366</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>728</width>
        <height>346</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="_2">
       <item>
        <layout class="QHBoxLayout" name="Layout_content"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
